apps/backend/src/common/services/user_service.py

```
@@ -0,0 +1,509 @@
+"""User service for authentication and user management."""
+
+import logging
+import secrets
```

Ruff (0.11.9)

`secrets` imported but unused

Remove unused import: `secrets`

(F401)